test/testDynPoint2D.jl

```diff
@@ -14,9 +14,9 @@ using Test
 global fg = initfg()
 
 # add two point locations
-global v0 = addVariable!(fg, :x0, DynPoint2(ut=0))
+global v0 = addVariable!(fg, :x0, DynPoint2(); nanosecondtime=0)
 
-global v1 = addVariable!(fg, :x1, DynPoint2(ut=1000_000))
+global v1 = addVariable!(fg, :x1, DynPoint2(ut=1000_000); nanosecondtime=10^9)
```

Comment:
Hi @Affie , just checking here again and wondering if you don't want to drop the `DynPoint(ut=1000000)`?  You should perhaps go as far as the rest of the code and just use the type `DynPoint2` here.  IIF will internally create the object when necessary.

Reply:
Hello, I left it in to trigger and test the deprecation on it. 
I'll open an issue on T vs T() if that is what you mean.

Reply:
https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/577